src/charm.py

```diff
@@ -329,7 +344,7 @@ def _generate_monitoring_service(self, enabled: bool = True) -> Dict[str, str]:
             command = "true"
             startup = "disabled"
         return {
-            "override": "merge",
+            "override": "replace",
```

Comment:
JFMI, how is it related to the current PR? Q: why do we switch to replace?

Reply:
It's not related, but while doing it, I noticed that with merge we append to the after list. So on every merge we duplicate the list of service dependencies,